### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by zonzujiro**
I'm not sure, but I think, that if we add some additional requirements to this task, it could be last. 🤔 

Also, I need some time to consider what additional links I can add to help students to understand what they must do.

Your opinion are welcome :)

### CODE REVIEW

File: test10.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+story with mobile devices.
+
+### Materials which will help you:
+
+* [Weather API](http://openweathermap.org/api) - _you can use this API for
+  getting data for your website_
+* [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL) - you can use
+  this for working with URL
+* [Manipulating the browser history](https://developer.mozilla.org/en-US/docs/Web/API/History_API) -
+  you must now how to work with browser history
```
Comment (sudodoki) [inline test10.md]: now -> know

File: test10.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+If you fill that you can do a better job, you can use
```
Comment (sudodoki) [inline test10.md]: fill -> feel

File: test10.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* enter name of the city or it's postcode and get weather cast in this city
+* choose period of the weather cast: today, tomorrow, week, two weeks
+* choose units — Celsius or Fahrenheit
+* have a list of favorite cities. By selecting favorite city from the list, I
+  want to see weather cast in it
+* have a list of recently viewed cities
+* see current city in the link in the address bar of the browser. If I open this
+  link in the new window, website must show me weather in this city
+* see fancy icons for weather cast and design. Using of template absolutely
```
Comment (sudodoki) [inline test10.md]: for some reason I feel like 
`Using of template` is somewhat strange. Can `Using template` or `Usage of template`  be a better fit, maybe?

**Comment by cidre** [comment]
May be I wrong, but 16 day forecast is available only for paid accounts, isn't it?

**Comment by sudodoki** [comment]
@cidre even if so, it should be enough for `today, tomorrow, week, two weeks` periods

**Comment by cidre** [comment]
@sudodoki Judging by their price list, only 5 days for free.

**Comment by sudodoki** [comment]
https://github.com/toddmotto/public-apis#weather might want to try some of these

**Comment by zonzujiro** [comment]
I changed the link to API with the link provided by @sudodoki and removed `two weeks` weather cast period from requirements.

Thanks, @cidre

**Comment by cidre** [comment]
I would replace "may vary from" -> "may depend on"

**Comment by cidre** [comment]
Sorry, may be some typos: "In case you trying..." -> "... try ..",  "...does not supports..." -> "... support ..." )

**Comment by zonzujiro** [comment]
I think this is final version

**Comment by cidre** [comment]
tornadoes ) 
Good job!

**Pull request merged**